src/frontend/src/App.vue

```diff
@@ -1,13 +1,102 @@
 <template>
   <div id="app">
+    {{ mining }}
     <router-view />
   </div>
 </template>
 
 <script lang="ts">
 import { Vue } from "vue-property-decorator";
+import { io } from "socket.io-client";
+import { sha256 } from "js-sha256";
+import type { Transaction } from "./types";
 
-export default class App extends Vue {}
+const socket = io("http://localhost");
```

Comment:
You can remove some of these extra lines and maybe write a comment above socket to have a defining area for variables outside of export default!